engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/views/register.html

```diff
@@ -15,17 +15,21 @@ <h2 translate>Content Host Registration</h2>
       </li>
       <li>
         <p translate>Install the pre-built bootstrap RPM:</p>
-        <pre><code>rpm -Uvh http://{{ noCapsulesFound ? katelloHostname : hostname(selectedCapsule.url) }}/pub/{{ consumerCertRPM }}</code></pre>
+        <pre><code>rpm -Uvh https://{{ noCapsulesFound ? katelloHostname : hostname(selectedCapsule.url) }}/pub/{{ consumerCertRPM }}</code></pre>
       </li>
       <li>
-        <p translate>Run subscription-manager in a console on the client host.  You may use an Activation Key to register:</p>
+        <p translate>Register using subscription-manager:</p>
+        <p translate>Using an Activation Key:</p>
         <pre><code>subscription-manager register --org="{{ organization.label }}" --activationkey="&ltActivation Key Name&gt"</code></pre>
-        <p translate>or authenticate with a username and password:</p>
+        <p translate>Using a username and password:</p>
         <pre><code>subscription-manager register --org="{{ organization.label }}" --environment="Library"</code></pre>
       </li>
       <li>
-        <p translate>Install katello-agent for remote actions and displaying errata information:</p>
+        <p translate>Install an Agent:</p>
```

Comment:
This seems like it could be dropped. Only one of the packages is actually an agent.

Reply:
"install client package" or do you have any suggestions? @ehelms

Reply:
That sounds good